src/state/StateKeyValue.cpp

```diff
@@ -236,7 +236,7 @@ void StateKeyValue::flagChunkDirty(long offset, long len)
 void StateKeyValue::markDirtyChunk(long offset, long len)
 {
     isDirty |= true;
-    memset(BYTES(dirtyMask) + offset, ONES_BITMASK, len);
+    memset(dirtyMask.get() + offset, ONES_BITMASK, len);
```

Comment:
I know this was like this before, but in general we're trying to use a leading `::` on functions from the global namespace to avoid confusion. No need to change these now (unless you want to), but just a note if coming across them in future. 

Another thing I looked for a `clang-format`/`-tidy` rule for but couldn't find.